workshops/personal_website/README.md

```diff
@@ -407,10 +407,8 @@ img {
 - **Select** every **`img` tag** on this HTML page and set the **width** to be
   **`200px`**
 
-> **`px`** stands for **pixels**. **Pixels** are the tiny dots that make up a
-> computer screen. By setting the width to **200px**, we are saying we want
-> every image on the page to span the length of 200 pixels (which is about 2
-> inches on a screen). If you want to learn more about pixels, you can Google
+> `px` stands for **pixels**, a unit of size on the screen. If you want to learn
+> more about pixels, you can Google search
 > ["What is a pixel?"](https://www.google.com/#q=what+is+a+pixel)
```

Comment:
I think we could probably pretty easily use some other measurement of size
(width of thumb or just centimeters).

On Wed, Feb 24, 2016 at 10:04 AM, Max Wofford <notifications@github.com>
wrote:

> In workshops/personal_website/README.md
> <https://github.com/hackclub/hackclub/pull/735#discussion_r53979408>:
>
> > @@ -407,10 +407,8 @@ img {
> >  - **Select** every **`img` tag** on this HTML page and set the **width** to be
> >    **`200px`**
> >
> > -> **`px`** stands for **pixels**. **Pixels** are the tiny dots that make up a
> > -> computer screen. By setting the width to **200px**, we are saying we want
> > -> every image on the page to span the length of 200 pixels (which is about 2
> > -> inches on a screen). If you want to learn more about pixels, you can Google
> > +> `px` stands for **pixels**, a unit of size on the screen. If you want to learn
> > +> more about pixels, you can Google search
> >  > ["What is a pixel?"](https://www.google.com/#q=what+is+a+pixel)
>
> @zachlatta <https://github.com/zachlatta> As for removing the example
> size (in inches), https://gist.github.com/remixz/3bf575b32bb96be43ea0
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/735/files#r53979408>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>